### ISSUE TITLE
how to setting rf model not boosting just bagging

### ISSUE THREAD

**Issue by weidong8405** [microsoft/LightGBM#2347]
i set the config like this
 boosting_type = rf
but the bagging num_trees how to set for, i do not know
not the num_trees which is to set for the boosting iteration

**Comment by guolinke** [comment]
sorry, I don't understand your question. 
`num_trees` in RF don't boost. the trees are independent.

**Comment by weidong8405** [comment]
but i read the code when generate a tree is will update the predict score use learning_rate, in RF the learning rate is useless?

**Comment by guolinke** [comment]
yeah, the learning rate, of course, is not used in RF model.

refer to https://github.com/microsoft/LightGBM/blob/0551f77517d971ce9bc32e2633768245957b410a/src/boosting/rf.hpp#L48

learning_rate is forcely set to 1 in RF.
